.husky/pre-commit

```diff
@@ -25,15 +25,40 @@ if [ "$HUSKY_PRE_COMMIT" != 0 ]; then
             exec git diff --cached --color | grep -nE $consoleregexp -A 2 -B 2
             read -p "There are some occurrences of forbidden patterns at your modification. Are you sure want to continue? (y/n)" yn
             echo $yn | grep ^[Yy]$
-            if [ $? -eq 0 ]
-            then
-                exit 0; #THE USER WANTS TO CONTINUE
-            else
+            if [ $? -ne 0 ]; then
                 exit 1; # THE USER DONT WANT TO CONTINUE SO ROLLBACK
             fi
         else
             echo -e "\033[1;31mWARNING: Some occurences of forbidden patterns were detected \033[0m"
             exec git diff --cached --color | grep -nE $consoleregexp -A 2 -B 2
         fi
     fi
+
+    # Css wrap check
+    scssregexp='.*\.scss'
+    ignoreFileInstruction='\/\* husky-ignore \*\/'
+    filelist=()
+    for file in $(git diff --cached --name-only | grep -E $scssregexp); do
```

Comment:
check is not enought :x This is not flagged as invalid css

![image](https://github.com/user-attachments/assets/ee00e3e3-8d05-4515-958f-2f6e8a85212b)

Reply:
I know, we discusswed this limitation yesterday? it's more about having a generic check so the dev stays alert rather than hafin a full parser just for the fun.. or else I need to make a js script because standard grep is not smart enough to get it done but it seemed not necessary. @LucasLefevre

Reply:
Grep can probably still do the job in this case I think. We could maybe have a regex `(startOfLine)(somethingThatIsn't.o-spreadshhet)` Will only work is the CSS is correctly prettified but w/e.

This should work ? 

`^(\.(?!o-spreadsheet)|[a-zA-Z]+).*\{`

Reply:
you need to handle multi line grep which does not work on macos for instance, so no it does not work.

Reply:
i'll try to do a correct js script whenever I have the time